locales/de/base.json

```diff
@@ -64,8 +65,7 @@
       "front": "Alle nach vorn bringen"
     },
     "system": {
-      "system": "System",
-      "enable-browser-access": "Browserzugriff aktivieren"
+      "enable-browser-access": ""
```

Comment:
We shouldn't have any empty values in the translations. If there's no transation we can remove the property altogether. Right @sallar ?

Reply:
I left the empty translations inside because a person who might want to translate something can see it better that way. If a field is empty, the english language is the fallback.

Reply:
Hmm.. should be a nicer way to do this other than leaving them blank. People won't do this themselves when adding new items, and the translators themselves handle non-existant values and then using the fallback instead.

Best to remove these for consistency. It's also what will be used for the mobile application's translations.

Reply:
Ok :) https://github.com/buttercup/buttercup-desktop/pull/883